docs/config/xmlConfigurationFiles.rst

```diff
@@ -128,17 +128,31 @@ The configuration value itself is defined as a cvalue element. Each element can
 * *type* - There exist several types for configuration values. The default value is string. Note, that the selection of
   the type will influence, how variables are interpreted and evaluated / converted.
 
-  - *string* will
+  - *string* accepts any value.
 
   - *int* will accept integer values only. E.g. 1, 2, 3 or 4.
 
-  - *float* will accept float values in differnt formats. E.g. 1.2f 1.2
+  - *float* will accept valid Java / Groovy float values. E.g. 1.2f 1.2
```

Comment:
I think the correct term here is "float literal", not "float value". 
I would remove Groovy. I think people will rather find the Java docs for literals, or not?